vault/logical_system.go

```diff
@@ -1662,6 +1663,13 @@ func (b *SystemBackend) handleMountTuneWrite(ctx context.Context, req *logical.R
 		return logical.ErrorResponse("missing path"), nil
 	}
 
+	// user_lockout configuration needs to be changed only with sudo capability
+	isSudo := b.Core.tokenStore.System().(extendedSystemView).SudoPrivilege(ctx, path, req.ClientToken)
+	userLockoutConfigMap := data.Get("user_lockout_config").(map[string]interface{})
```

Comment:
Do we need to use `.GetOk()` here? What happens if the `user_lockout_config` parameter isn't passed to this request?

Reply:
Get() uses GetOK() internally. Here, if "user_lockout_config" is not passed, value is not added to the map (userLockoutConfigMap). The length of the map remains 0. It does not panic.